### PR TITLE
NMA-303 Fund New Account | Confirm Transaction

### PR DESCRIPTION
**Pull request by tomasz-ludek**

### CODE REVIEW

File: wallet/src/de/schildbach/wallet/ui/WalletActivity.java [hunk 1]
```diff
@@ -295,7 +295,8 @@ private void showHideJoinDashPayAction() {
         final Coin walletBalance = wallet.getBalance(Wallet.BalanceType.ESTIMATED);
         boolean canAffordIt = walletBalance.isGreaterThan(Constants.DASH_PAY_FEE)
                 || walletBalance.equals(Constants.DASH_PAY_FEE);
-        boolean visible = syncComplete && canAffordIt && config.getShowJoinDashPay();
+        //FIXME to be fixed after review
+        boolean visible = true ;//syncComplete && canAffordIt && config.getShowJoinDashPay();
```
Comment (tomasz-ludek) [inline wallet/src/de/schildbach/wallet/ui/WalletActivity.java]: This was changed to make testing easier and **needs to be changed back before merging.**

**Comment by HashEngineering** [comment]
Does this get merged before or after NMA-302?

**Comment by tomasz-ludek** [comment]
@HashEngineering ,
this is supposed to be merged **after** https://github.com/dashevo/dash-wallet/pull/342 (NMA-302), I set `join-dashpay-quick-action` as a base branch just to show only the related commits in PR. I will change it to `evonet` as soon as https://github.com/dashevo/dash-wallet/pull/342 is merged.

File: wallet/src/de/schildbach/wallet/ui/WalletActivity.java (hunk 1 above)
Comment (HashEngineering) [reply wallet/src/de/schildbach/wallet/ui/WalletActivity.java]: We didn't change this, but can change it later.

**Comment by HashEngineering** [comment]
This should have been merged before #347, but wasn't.  It is not needed anymore since the code is part of #347.

**Pull request closed without merging**